core/storage/usermanagement.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package storage defines storage interfaces.
+package storage
+
+import (
+	"context"
+
+	tpb "github.com/google/keytransparency/core/api/type/type_proto"
+)
+
+// KeySets represents the interface for the usermanager.
```

Comment:
Do you think a `usermanager` will implement other interfaces as well?

Reply:
Perhaps this should be keyset storage.